python/python/bystro/ancestry/listener.py

```diff
@@ -28,88 +24,6 @@
 logger = logging.getLogger()
 
 ANCESTRY_TUBE = "ancestry"
-ANCESTRY_BUCKET = "bystro-ancestry"
```

Comment:
all of the ancestry model loading was moved to a dedicated module so that it could be cleanly shared in the ancestry api module, without having to load ray and beanstalkd (this allows us to not have to ray start --head before running  ancestry cli commands)